jit.en.vtt

```diff
@@ -432,7 +432,7 @@ call them, micro-ops), but the more
 complicated ones, like FOR_ITER_RANGE or
 
 05:38.319 --> 05:40.240 align:start position:0%
-the integer addition, may have several
+the integer addition, many have several
```

Comment:
I think “may” is correct here.

Reply:
Definitely sounds like "many" to my ear.

Some might expand to only two parts, some might expand to many.

The examples given are `FOR_ITER_RANGE` and integer addition (`BINARY_OP_ADD_INT`?), which expand to many:

<img width="865" alt="image" src="https://github.com/encukou/subtitles-for-brandts-talk/assets/1324225/31991255-7ada-401d-8054-d847d2e3b81a">

<img width="845" alt="image" src="https://github.com/encukou/subtitles-for-brandts-talk/assets/1324225/bf637ed9-c451-423a-b66f-f99f4b51b6a0">

https://youtu.be/HxSHIpEQRjs?si=Wl9JoffMA0jHsj-k&t=332

Reply:
I agree that it sounds like "many" in the video. But I said "may". :)

Reply:
```suggestion
the integer addition, may have several
```